src/main/java/edu/depaul/cleanSweep/controlSystem/Cleaner.java

```diff
@@ -177,8 +179,158 @@ private boolean checkNodeCleanAndVisited(FloorTile sensor) {
 		//System.out.println("^^^^^^^^^heading"+(char) dest[0]); //for test
 		return new int[] {ch, ax, ay};
 	}
-	
-	
+
+	//does not consider the original heading towards
+	public void move2Adjacent(int bx, int by) {
```

Comment:
How come we needed a different moving function?

Reply:
modified as requested.